lms/djangoapps/courseware/tests/test_discussion_xblock.py

```diff
@@ -255,7 +255,7 @@ def test_student_perms_are_correct(self, permissions):
 @ddt.ddt
 class TestXBlockInCourse(EnableTransformerRegistryMixin, SharedModuleStoreTestCase):
     """
-    Test the discussion xblock data returned by Course API.
+    Test the discussion xblock as rendered in the course and course API.
```

Comment:
Sorry, meant to include this in the previous commit.  It'll all get squashed in the wash eventually anyway :)